jobs/openshift4/cluster/create/Jenkinsfile

```diff
@@ -106,8 +106,13 @@ node("cirhos_rhel7") {
                         sh "./scripts/dedicated-setup.sh"
                     }
                 } else {
+                    build (job: 'openshift4-cluster-deprovision', parameters:[
+                        [$class: 'StringParameterValue', name: 'CLUSTER_NAME', value: CLUSTER_NAME],
+                        [$class: 'StringParameterValue', name: 'RECIPIENTS', value: RECIPIENTS],
+                        [$class: 'StringParameterValue', name: 'LOG_LEVEL', value: LOG_LEVEL],
+                        [$class: 'StringParameterValue', name: 'CREDENTIALS_ID', value: CREDENTIALS_ID]
+                    ])
```

Comment:
I think this is fine, we want to just destroy the cluster if it cant install it ... maybe. Can go with it for now anyway.